hack/trusted-tasks-update.sh

```diff
@@ -10,25 +10,48 @@
 # be controlled by `$COLLECT` which can be set to `git` or `oci`
 #
 # Parameters via environment variables:
-# COLLECT        - can be set to `oci` or `git` (defaults to both), what to
-#                  resolve
-# INPUT_IMAGE    - Conftest OCI bundle containing the trusted tasks list, defaults
-#                  to:
-#                  `quay.io/redhat-appstudio-tekton-catalog/data-acceptable-bundles:latest`
-# OUTPUT_IMAGE   - Conftest OCI bundle to write the trusted task list to, defaults
-#                  to `$INPUT_IMAGE`
-# QUAY_NAMESPACE - Quay namespace to query for Task bundles (defaults to
-#                  `redhat-appstudio-tekton-catalog`)
+# COLLECT         - can be set to `oci` or `git` (defaults to both), what to
+#                   resolve
+# INPUT_IMAGE     - Conftest OCI bundle containing the trusted tasks list, defaults
+#                   to:
+#                   `quay.io/redhat-appstudio-tekton-catalog/data-acceptable-bundles:latest`
+# OUTPUT_IMAGE    - Conftest OCI bundle to write the trusted task list to, defaults
+#                   to `$INPUT_IMAGE`
+# QUAY_NAMESPACES - Quay namespaces to query for Task bundles (defaults to
+#                   `redhat-appstudio-tekton-catalog konflux-ci/tekton-catalog`)
 
 set -o errexit
 set -o nounset
 set -o pipefail
 
 mapfile -td ' ' COLLECT < <(echo -n "${COLLECT:-git oci}")
-INPUT_IMAGE=${INPUT_IMAGE:-quay.io/redhat-appstudio-tekton-catalog/data-acceptable-bundles:latest}
+mapfile -td ' ' QUAY_NAMESPACES < <(
+    echo -n "${QUAY_NAMESPACES:-'redhat-appstudio-tekton-catalog konflux-ci/tekton-catalog'}"
+)
+
+INPUT_IMAGE=${INPUT_IMAGE:-quay.io/konflux-ci/tekton-catalog/data-acceptable-bundles:latest}
```

Comment:
So this image is a bit unusual. We don't build it from scratch every time. Instead, this image gets incremental updates, which means the existing data matters.

When we have a new location for this image, we should "seed" it with some relevant data so it contains the full set of expected data. There are different ways of doing this.

1. Change the input image to always be the existing one from `redhat-appstudio-tekton-catalog`. As a follow up PR, we can switch to the new location.
2. Copy the existing image from `redhat-appstudio-tekton-catalog` to `konflux-ci` just before merging this PR.

Thinking about this further, there's still room for deviation between the two images. An alternative is to simply add a skopeo copy command at the end of the script to copy the image from `redhat-appstudio-tekton-catalog` to `konflux-ci`. At some point in the future, we just update the script to just maintain the `konflux-ci` one.

WDYT?

Reply:
> Copy the existing image from redhat-appstudio-tekton-catalog to konflux-ci just before merging this PR.

:+1: This was the plan

> Thinking about this further, there's still room for deviation between the two images. An alternative is to simply add a skopeo copy command at the end of the script to copy the image from redhat-appstudio-tekton-catalog to konflux-ci. At some point in the future, we just update the script to just maintain the konflux-ci one.

Good point. I don't think I can skopeo copy at the end of the script (the credentials for pushing to redhat-appstudio-tekton-catalog are in a separate secret, `tkn bundle push` didn't like multiple `quay.io/...` auths in a single file). But I can probably skopeo copy in a separate step in the pipeline

Reply:
Just to note that this script is not part of the CI, we call it as needed when we miss an update to make sure to include it, so we can merge it as is.

Reply:
Oh, TBH I didn't realize that we were talking about this script. The same applies to .tekton/scripts/build-acceptable-bundles.sh, so I updated the push pipeline to just `skopeo copy` the bundle from redhat-appstudio-tekton-catalog to konflux-ci/tekton-catalog

Reply:
re-tested, now the bundles are more obviously identical which is nice
* https://quay.io/repository/acmiel-test/tekton-catalog/data-acceptable-bundles?tab=tags&tag=latest
* https://quay.io/repository/acmiel-rhtap/data-acceptable-bundles?tab=tags&tag=latest